docs/apis/apis.md

```diff
@@ -26,12 +28,19 @@ See [packaging models](packaging-models.md) for how to create the zipped model.
 - kind: api
   name: my-api
   model: s3://my-bucket/my-model.zip
+  request_handler: inference.py
   compute:
     min_replicas: 5
     max_replicas: 20
     cpu: "1"
 ```
 
+## Custom Request Handlers
+
+Request handlers are used to decouple the interface of an API endpoint from its model. A `pre_inference` request handler can be used to modify request payloads before they is sent to the model. A `post_inference` request handler can be used to modify model predictions before they are served from the API endpoint.
```

Comment:
nit: "before they are served from the API endpoint" -> "in the server before they are sent to the client"